test/test-network/systemd-networkd-tests.py

```diff
@@ -101,7 +102,7 @@ def f(func):
         support = False
         rc = call('ip link add name dummy99 type dummy')
         if rc == 0:
-            ret = run('udevadm info -w10s /sys/class/net/dummy99', stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+            ret = run(*udevadm_cmd, 'info -w10s /sys/class/net/dummy99', stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
I've tried this, and unfortunately this does not work... This is why I couldn't submit to revert the commit.

Reply:
I see, it indeed does not (and it also looks like the sanitizers run just got stuck with this change). I guess it would make sense to just defer this PR.